src/components/atoms/DetailTop/DetailTop.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image"
+
+interface DetailTopProps {
+  name: string // シェフ名 or 料理名
+  content: string // 説明文
+  url: string // 画像URL
+  category: "chef" | "recipe"
+}
+
+/**
+ * 詳細画面(シェフor料理)の上部に表示するコンポーネント
+ */
+export const DetailTop = ({ name, content, url, category }: DetailTopProps) => {
+  return (
+    <div className="flex flex-col max-w-[390px]">
+      <div
+        className="mb-4"
+        style={{
+          position: "relative",
+          width: "390px",
+          height: "390px",
+        }}
+      >
+        <Image src={url} fill style={{ objectFit: "cover" }} alt="TopImage" />
+      </div>
+      <p className="text-Mauve-12 text-fs22 px-4 mb-3 font-bold">{name}</p>
+      <p className="text-Mauve-12 text-fs16 px-4 mb-4 font-normal">{content}</p>
+    </div>
+  )
```

Comment:
以下お願いいたします 🙇 
- styleで記述しているもの→TailwindCSSで書く
- 全体のmax-wを390px→480pxに変更（390px以上のスマホで表示が崩れるため）
- 不要なスタイル削除
- padding-top100%で縦横比を1:1で維持

コメントだけだと伝わりにくいと思うので、こんな感じかなー？っていうのを載せておきます！
```suggestion
  return (
    <div className="max-w-[480px]">
      <div className="mb-4 relative pt-[100%]">
        <Image src={url} fill className="object-cover" alt="TopImage" />
      </div>
      <p className="text-Mauve-12 text-fs22 px-4 mb-3 font-bold">{name}</p>
      <p className="text-Mauve-12 text-fs16 px-4 mb-4 font-normal">{content}</p>
    </div>
  )
```

Reply:
修正しました。
比率の設定はaspect-ratio使いました！